stdlib/2and3/logging/__init__.pyi

```diff
@@ -275,7 +275,7 @@ else:
     @overload
     def getLogger() -> Logger: ...
     @overload
-    def getLogger(name: str) -> Logger: ...
+    def getLogger(name: Union[unicode, str]) -> Logger: ...
```

Comment:
This should be `Union[Text, str]` -- in Python 3 mode there's no `unicode`. (Though I'm confused why the tests didn't catch this.)